README.md

```diff
@@ -4,7 +4,131 @@
 - [GoDoc](https://godoc.org/go.linecorp.com/centraldogma)
 - [CLI manual](https://line.github.io/centraldogma/client-cli.html)
 
-## How to build
+## Usage
+
+### Getting started
+
+```go
+import "go.linecorp.com/centraldogma"
+
+// create client with OAuth2 token
+centraldogma.NewClientWithToken(baseURL, token, nil)
+```
+
+### Customize transport
+
+If transport is `nil` (like above), `http2.Transport` is used by default. You could inject your own transport easily:
+
+```go
+import "golang.org/x/net/http2"
+
+tr := &http2.Transport{
+    DisableCompression: false,
+}
+
+// create client
+centraldogma.NewClientWithToken(baseURL, token, tr)
+```
+
+### Example
+
+```go
+package sample
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"go.linecorp.com/centraldogma"
+)
+
+// CentralDogmaFile represents a file in central dogma.
+type CentralDogmaFile struct {
+	client            atomic.Value
+	BaseURL           string       `yaml:"base_url" json:"base_url"`
+	Token             string       `yaml:"token" json:"token"`
+	Project           string       `yaml:"project" json:"project"`
+	Repo              string       `yaml:"repo" json:"repo"`
+	Path              string       `yaml:"path" json:"path"`
```

Comment:
How about removing `Path` and let the user specify it as an argument in `Fetch()` and `Watch()`?

Reply:
This is example to manage (watch/fetch content) of single file entry. So user could specific `Path` from configuration file. So it's not necessary to pass it as argument here.

Reply:
Yeah, I was thinking that maybe we could change this example a little bit to
something like more realistic. (i.e. Change `CentralDogmaFile` to `CentralDogmaRepo` so that people can use it with a different path by specifying it as an argument.) But like you said, if you want to just show how to manage a single file entry, that's fine.
Either way is fine to me. :)